app/plugins/cron.js

```diff
@@ -0,0 +1,14 @@
+const cron = require('node-cron')
+const processComplianceApplications = require('../messaging/application/process-compliance-applications')
+const { complianceScheduler } = require('../config')
+
+module.exports = {
+  plugin: {
+    name: 'processComplianceApplications',
```

Comment:
Please consider renaming the file to process-compliance-application.js as it contains compliance specific logic.